calico/about/about-calico.mdx

```diff
@@ -42,7 +42,7 @@ The following table summarizes the main {{prodname}} features. To search for spe
 | Feature                      | Description                                                  |
 | ---------------------------- | ------------------------------------------------------------ |
 | Dataplanes                   | eBPF, standard Linux iptables, Windows HNS, VPP.         |
-| Networking                   | &bull; Scalable pod networking using BPG or overlay networking<br />&bull; Advanced IP address management that is customizable |
+| Networking                   | &bull; Scalable pod networking using BGPor overlay networking<br />&bull; Advanced IP address management that is customizable |
```

Comment:
```suggestion
| Networking                   | &bull; Scalable pod networking using BGP or overlay networking<br />&bull; Advanced IP address management that is customizable |
```

Reply:
@ctauchen 
I fixed that typo and also fixed "iBGProuter" to "iBGP router" and push it.